src/components/App/App.tsx

```diff
@@ -29,19 +32,30 @@ export const App = () => {
   }, []);
 
   const toggleRecording = () => {
-    if (currentRecordingStatus === 'recording')
-      return setCurrentRecordingStatus('recorded');
-    return setCurrentRecordingStatus('recording');
+    return currentRecordingStatus === 'recording'
+      ? setCurrentRecordingStatus('recorded')
+      : setCurrentRecordingStatus('recording');
```

Comment:
If a better name for `setCurrentRecordingStatus` can be found that enables the ternary to go onto one line, great. Otherwise please revert this change.